README.md

```diff
@@ -62,24 +64,27 @@ Accepted params:
 ```php
 require __DIR__ . '/vendor/autoload.php';
 
-use Auth0\SDK\API\Authentication;
+use Auth0\SDK\Auth0;
```

Comment:
I agree it is confusing but I should have done at the very beginning. hope it is the last time I change my mind